fitbenchmarking/results_processing/tests/test_base_table.py

```diff
@@ -230,3 +233,21 @@ def test_display_str_both(self):
         self.table.options.comparison_mode = 'both'
         s = self.table.display_str([7, 9])
         self.assertEqual(s, '9 (7)')
+
+    @mock.patch("fitbenchmarking.results_processing.base_table.plt.savefig")
+    def test_save_colourbar_returns_a_relative_path(self, savefig_mock):
+        """
+        Test the relative path to the colourbar figure is returned after saving
+        """
+        self.table.name = "fake_name"
+        colourbar_file = f"{self.table.name}_cbar.png"
+
+        figure_sub_directory = "fitbenchmarking_results"
+        figure_directory = os.path.join(self.root_directory,
+                                        figure_sub_directory)
+
+        relative_path = f"{figure_sub_directory}/{colourbar_file}"
```

Comment:
I think os.path.join here will remove the need for the replace in the check at the end?

Reply:
Thanks, I think that works much better!